dragome-web/src/main/java/com/dragome/web/config/DomHandlerApplicationConfigurator.java

```diff
@@ -112,6 +112,6 @@ public DomHandlerApplicationConfigurator(List<? extends Class<?>> additionalDele
 	}
 	private void createJsDelegateGenerator(ClassPath classpath)
 	{
-		jsDelegateGenerator= new JsDelegateGenerator(classpath.toString().replace(";", ":"), new DomHandlerDelegateStrategy());
+		jsDelegateGenerator= new JsDelegateGenerator(classpath.toString(), new DomHandlerDelegateStrategy());
```

Comment:
Hang on, I'll replace any semicolon with path separator system property and
see if that works on my side.

On Thu, Apr 7, 2016 at 4:16 PM, Fernando Petrola <notifications@github.com>
wrote:

> In
> dragome-web/src/main/java/com/dragome/web/config/DomHandlerApplicationConfigurator.java
> <https://github.com/dragome/dragome-sdk/pull/91#discussion_r58878332>:
>
> > @@ -112,6 +112,6 @@ public DomHandlerApplicationConfigurator(List<? extends Class<?>> additionalDele
> >  	}
> >  	private void createJsDelegateGenerator(ClassPath classpath)
> >  	{
> > -		jsDelegateGenerator= new JsDelegateGenerator(classpath.toString().replace(";", ":"), new DomHandlerDelegateStrategy());
> > +		jsDelegateGenerator= new JsDelegateGenerator(classpath.toString(), new DomHandlerDelegateStrategy());
>
> It's ok for windows but I guess it will crash in *nix now
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/dragome/dragome-sdk/pull/91/files/78b57bb75c8afaaf33a78469a0fc8ad2ef5136f7#r58878332>
>